src/packages/cli/src/cli.ts

```diff
@@ -8,6 +8,17 @@ import initializeEthereum from "./initialize/ethereum";
 import initializeFilecoin from "./initialize/filecoin";
 import type { FilecoinProvider } from "@ganache/filecoin";
 import type { EthereumProvider } from "@ganache/ethereum";
+import {
+  notifyDetachedInstanceReady,
+  stopDetachedInstance,
+  startDetachedInstance,
+  getDetachedInstances
+} from "./detach";
+import { TruffleColors } from "@ganache/colors";
+import { table } from "table";
```

Comment:
Yeah - it's good enough! I have used `table` in some tools I've written in the past, but not for anything complex.

You triggered me to look back into this, and have switched out `table` for `cli-table`, as that's an existing dependency via `marked-terminal`. The API is almost the same.

I've had to modify the styling, because by default the headers are red 🤮, but it looks mostly indistinguishable now.